streamparser.py

```diff
@@ -158,7 +159,15 @@ def __init__(self, lexical_unit):  # type: (str) -> None
         self.lexical_unit = lexical_unit
 
         cohort = re.split(r'(?<!\\)/', lexical_unit)
-        self.wordform = cohort[0]
+
+        if("]]^" in cohort[0]):
+            split_form = cohort[0].split("]]^")
```

Comment:
Do you want to limit this split? i.e. 
```suggestion
            split_form = cohort[0].split("]]^", 1)
```

Reply:
sure, although ]]^ cant occur in it more than once, but doesn't hurt